tasks/section_3_Network_Configuration.yaml

```diff
@@ -530,7 +530,16 @@
         name: ufw
         state: started
         enabled: true
-      when: UFWEnable
+    
+    - name: 3.5.1.3 Ensure ufw service is enabled | If needed, reset any rules manually added to UFW before enable
+      shell: 
+        cmd: "ufw --force reset" 
+        # notes:
+        # port 22 is going to be allowed in the next task (also in 3.5.1.6, btw)
```

Comment:
We already checked that this won't disconnect the current ansiblke session, right?

Reply:
According to the ufw documentation and also to the comments in lines 521-525 of this file, ufw supports adding rules without being enabled. This means that as long as ufw is not enabled, it will not enforce any disconnections based on the ports it allows or disallows. 

I'll add a previous step to disable ufw before executing the step 3.5.1.3, just to be sure that ufw was not previously manually enabled.

```
# 3.5.1.3 Ensure ufw service is enabled
# When running ufw enable or starting ufw via its initscript, ufw will flush its chains.
# This is required so ufw can maintain a consistent state, but it may drop existing
# connections (eg ssh). ufw does support adding rules before enabling the firewall.
# Run the following command before running ufw enable
```